release-notes/swan-lake-alpha5-release-note.md

```diff
@@ -37,8 +37,62 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
+
+##### `lang.xml:getDescendants()` Lang Library Function
+
+The `lang.xml:getDescendants()` lang library function has been introduced to retrieve a sequence of the descendant elements for a given element, in document order.
+
+##### Bitwise Operations as Constant Expressions
+
+Bitwise binary operations `<<`, `>>`, `>>>`, `^`, `&`, `|` are now allowed as constant expressions.
+
+```ballerina
+const BASE = 0x10;
+const int SHIFTED = (BASE << 2) | 0xa;
+```
+
 #### Improvements
 
+##### Improvements to the XML Literal
+
+XML template literals now support the inclusion of multiple XML items of the same `xml` subtype as well as different `xml` subtypes.
+
+```ballerina
+xml x1 = xml `<ele>item</ele><!--comment-->text<?foo?>`;
+xml<xml:Element> x2 = xml `<ele>item1</ele><ele>item2</ele>`;
+xml<xml:Comment> x3 = xml `<!--comment1--><!--comment2-->`;
+xml<xml:ProcessingInstruction> x4 = xml `<?foo?><?bar?>`;
+string interpolation = "Interpolation";
+xml<xml:Text> x5 = xml `text ${interpolation}`;
+```
+
+##### Normalization of XML Attribute Values
+In XML template literals, character references within XML attribute values are now normalized to the referring character. For example, the attribute value `e.attr` in the following example is now `a&b`.
+
+```ballerina
+xml:Element e = xml `<elem attr="a&amp;b" />`;
+```
+
+##### Breaking Changes
+- The `lang.array:lastIndexOf` and `lang.array:indexOf` lang library functions can only be used with arrays of which the element types are subtypes of the `anydata` type.
+- A compilation error occurs if the field visibility qualifiers in the including type and the included type with object type inclusion are not the same.
+- Remote method call action cannot be used to call non-remote methods.
+- The `xml` and `readonly` type-descriptors can no longer be used as expressions. A reference to a type definition defined with such type-descriptors needs to be used instead.
+- The argument passed to the second parameter of the `attach` method of the listener for service declarations has been fixed as follows. If there is no attach point in the service `()` (nil) is passed as the argument. If the attach point is the root resource path (“/”) an empty array is passed.
+- The `lang.xml:getContent()` function no longer accepts `xml:Text` as an argument. The new `lang.xml:data()` function can be used instead.
+- `lang.value` functions cannot be called on expressions of type `object`. Function call syntax (`value:func(obj)`) can be used instead.
+- The `lang.stream:reduce()` lang library function will only return the reduced value or an error (if the completion type includes error). It no longer returns a nil `()` value. The return type of `lang.stream:reduce()` has been updated accordingly.
+- An issue where the complement of numeric literals returned incorrect values has been fixed.
```

Comment:
```suggestion
- The issue of the complement of numeric literals returning incorrect values has been fixed.
```

Reply:
What we trying to say is that there was issue in this particular feature that has now been fixed. 

When we say "The issue of ...", IMO it kinda implies that we're talking about a specific issue that the user knows. Applicable to the next few points too.

Reply:
How about
```suggestion
- An issue with the complement of numeric literals where incorrect values were returned has been fixed.
```

Reply:
Although in github all of these are issues. But we use the topic **Bug Fixes** in the release note. IMO we can instead say *bug* instead of *issue*

Reply:
It doesn't sound correct to say "An issue". If we want to specify should say "the issue of...".

If not, since we are not linking to the issue also, we can just mention the fix without emphasizing whether it is a bug or an issue. For example,

"The complement of numeric literals do not return incorrect values anymore".

Reply:
IMO, that is too generic, we need to say that there was a specific issue in this particular scenario.

Why can't we say "An issue with the complement of numeric literals"?

There can be multiple issues with "complement of numeric literals", this is one particular issue.

Reply:
I like the idea of not emphasizing whether an issue or a bug. But that is better if these are under **Bug Fixes** section.
Problem with stating that "do not return incorrect values anymore" is we are capturing a very large scope, but technically we have only looked at one particular issue related to the area.

Reply:
+1 to have it like that under bug fixes.

Reply:
But like I mentioned previously, the reason why we are explicitly talking about them is because they are breaking changes. 

What we are trying to convey here is that the values returned may now change.

Reply:
Also, most of the changes here are bug fixes.

Reply:
How about

```suggestion
- A fix for a bug related to the complement of numeric literals may cause the result to change.
```

Reply:
I think it is better to keep it as it is. 
 > may cause the result to change
 This wording may make the change untrustworthy to the user.

Reply:
```suggestion
- An issue causing the complement of numeric literals to return incorrect values has been fixed.
```
@praneesha @MaryamZi